language-plutus-core/src/Language/PlutusCore/Lexer/Type.hs

```diff
@@ -89,11 +91,19 @@ which has the desired type signature:
     succInteger : forall s. integer s -> integer s
 -}
 
--- | The type of dynamic builtin functions. I.e. functions that exist on certain chains and do
+-- | The type of dynamic built-in types. I.e. types that exist on certain chains and do
+-- not exist on others. Each 'DynamicBuiltinType' has an associated kind --
+-- this allows to kind check dynamic built-in types just like static ones.
```

Comment:
This is confusing - reading this comment I expected there to be a kind here, but there isn't.

Reply:
And, indeed, given your note this seems to be false and they all have kind `*` (and that's what the typechecker says).

Reply:
Oh yes, sorry, old comment.